DNN Platform/Admin Modules/Dnn.Modules.Console/web.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<configuration>
+	<runtime>
+		<assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+			<dependentAssembly>
+				<assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30AD4FE6B2A6AEED" culture="neutral"/>
+				<bindingRedirect oldVersion="0.0.0.0-10.0.0.0" newVersion="10.0.0.0"/>
```

Comment:
No harm in having binding redirects but I looked at the projects and it turns out that we only have version 10 installed of Newtonsoft.Json. Why are there warnings in the first place?

Reply:
this warning should be caused by there have old Newtonsoft.Json assembly installed in system level.